meta-resin-common/recipes-containers/hostapp-update/files/hostapp-update

```diff
@@ -74,7 +74,7 @@ sync -f "$SYSROOT"
 
 if [ "$hooks" = 1 ]; then
 	# Run the defined hooks in the host OS we update to
-	if balena run --privileged --rm -v /mnt:/mnt -e DURING_UPDATE=1 "$HOSTAPP_IMAGE" hostapp-update-hooks; then
```

Comment:
The missing is on purpose?

Reply:
It defaults to the commit subject and since I find it descriptive enought, I left it as it is.